entrypoint.sh

```diff
@@ -45,15 +45,15 @@ if [[ "${GIT_SSH_PRIVATE_KEY}" != "" ]]; then
     chmod 600 ~/.ssh/id_rsa
     if [[ "${GIT_SSH_KNOWN_HOSTS}" != "" ]]; then
       echo "${GIT_SSH_KNOWN_HOSTS}" > ~/.ssh/known_hosts
-      git config --global core.sshCommand "ssh -i ~/.ssh/id_rsa -o IdentitiesOnly=yes -o UserKnownHostsFile=~/.ssh/known_hosts"
+      git config --global core.sshCommand "ssh -i /github/home/.ssh/id_rsa -o IdentitiesOnly=yes -o UserKnownHostsFile=/github/home/.ssh/known_hosts"
```

Comment:
👋 New user of the action here and running into this issue.

To simplify things, I would suggest _not setting_ the `-i` and `-o UserKnownHostsFile` arguments at all. The action is already writing the key and known_hosts to the default locations, so the default settings should be sufficient.

```suggestion
      git config --global core.sshCommand "ssh -o IdentitiesOnly=yes"
```

Of course, to ignore host fingerprints, you will still set `StrictHostKeyChecking` below:
```
        git config --global core.sshCommand "ssh -o IdentitiesOnly=yes -o StrictHostKeyChecking=no"
```

In both cases, you should be able to drop the `-i` and `-o UserKnownHostsFile`.

Reply:
Update: In addition, you don't need the `-o IdentitiesOnly=yes` option either. So, you'd _only_ need to define `core.sshCommand` in the case where you want to ignore fingerprints.